OMEdit/OMEditGUI/Animation/AnimationUtil.h

```diff
@@ -89,8 +89,9 @@ inline bool isCSV(const std::string& fileIn){
  */
 inline std::string assembleXMLFileName(const std::string& modelFile, const std::string& path){
   QString fileName(modelFile.c_str());
-  QRegExp fileTypeRegExp("(\\.fmu|\\.mat|\\.csv|_res.mat|_res.csv)");
-  fileName.remove(fileTypeRegExp);
+  QRegExp fileTypeRegExp("(_res.mat|_res.csv|\\.fmu)");
```

Comment:
I would suggest using QFileInfo (http://doc.qt.io/qt-4.8/qfileinfo.html). You can use the `QFileInfo.suffix` method to check what is the file extension and `QFileInfo.basename` to get the filename without extension. The use `QString.endswith` to check if it ends with _res.